{"dfs.nameservices": ["lklcluster"], "dfs.namenode.http-address.lklcluster.nn2": ["hadoop3:50070"], "dfs.datanode.data.dir": ["/home/hadoop/datadir2"], "dfs.ha.fencing.methods": ["sshfence"], "dfs.ha.fencing.ssh.connect-timeout": ["5000"], "dfs.ha.namenodes.lklcluster": ["nn1,nn2"], "hadoop.tmp.dir": ["/home/hadoop/tmp2"], "dfs.namenode.rpc-address.lklcluster.nn2": ["hadoop3:8020"], "dfs.namenode.rpc-address.lklcluster.nn1": ["hadoop2:8020"], "dfs.journalnode.edits.dir": ["/home/hadoop/journal/data"], "dfs.namenode.http-address.lklcluster.nn1": ["hadoop2:50070"], "dfs.namenode.name.dir": ["/home/hadoop/namedir2"], "dfs.namenode.shared.edits.dir": ["qjournal://hadoop1:8485;hadoop2:8485;hadoop3:8485/lklcluster"], "ha.zookeeper.quorum": ["hadoop1:2181,hadoop2:2181,hadoop3:2181"], "dfs.client.failover.proxy.provider.lklcluster": ["org.apache.hadoop.hdfs.server.namenode.ha.ConfiguredFailoverProxyProvider"], "dfs.ha.automatic-failover.enabled": ["true"], "fs.defaultFS": ["hdfs://lklcluster"], "dfs.ha.fencing.ssh.private-key-files": ["/home/hadoop/.ssh/id_rsa"]}